src/tasks/boozedrop.ts

```diff
@@ -90,13 +91,14 @@ export const BoozeDropQuest: Quest = {
           use($item`photocopied monster`);
         }
       },
-      outfit: {
+      outfit: () => ({
         back: $item`vampyric cloake`,
         weapon: $item`Fourth of May Cosplay Saber`,
         offhand: $item`industrial fire extinguisher`,
         familiar: $familiar`Cookbookbat`,
         modifier: "myst",
-      },
+        avoid: sugarItemsAboutToBreak(),
```

Comment:
Any reason why we are worried about the sugar items breaking after leveling? Does it cause that same annoying pop-up/crash like it does with the garbage shirt running out of charges?

Reply:
No crash, I just want to preserve them for the +25% stats for the stat tests, while also using them during leveling to boost our scalers.

Reply:
I'm not sure I understand - we run the stat tests before we run any other tests. So why are we avoiding their usage the these other tests (which occur after the stat tests)?

Reply:
But you're avoiding the equipping of sugar items on the spell (chapeau) and weapon (shank) tests too.

Also, I'm reading from the wiki that the sugar equips break on combats - have we confirmed that it also breaks on the tests? If so, I suppose the consideration is fine, but do remember to remove `avoid` on the tests you actually want to use the equips on

Reply:
Oh, if you look at the diff again, we're definitely keeping them for the actual tests. I pulled them into these files for some of the outfits in the pre-test adventures (e.g. ungulith here for the item test, when getting meteor showered in the spell damage test).